pages/linux/http_load.md

```diff
@@ -0,0 +1,21 @@
+# http_load
+
+> A useful HTTP benchmarking tool.
+> That lets you run multiple http fetches in parallel to test the throughput of web server.
+> More information: <http://www.acme.com/software/http_load/>.
+
+- Emulate 20 HTTP GET requests per seconds for 60 seconds:
+
+`http_load -rate {{20}} -seconds {{60}} urls.txt`
```

Comment:
Shouldn't urls.txt be a parameter?

Reply:
Sorry, urls.txt is a text file as a parameter. I fixed it.